containers/tefca-viewer/e2e/load.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from "@playwright/test";
+import { TEST_URL } from "../playwright-setup";
+import { metadata } from "@/app/constants";
+
+test("landing page loads", async ({ page }) => {
```

Comment:
this whole page is just a copy-paste from the existing file to help parallelize the playwright runners